spanner/api/SpannerTest/Tests.cs

```diff
@@ -133,13 +133,81 @@ public SpannerTests(SpannerFixture fixture)
                 if (!_fixture.s_initializedDatabase)
                 {
                     _fixture.s_initializedDatabase = true;
+                    DeleteStaleBackupsAndDatabases();
                     InitializeInstance();
                     InitializeDatabase();
                     InitializeBackup();
                 }
             }
         }
 
+        void DeleteStaleBackupsAndDatabases()
+        {
+            DatabaseAdminClient databaseAdminClient = DatabaseAdminClient.Create();
+            var instanceName = InstanceName.FromProjectInstance(_fixture.ProjectId, _fixture.InstanceId);
+            var databases = databaseAdminClient.ListDatabases(instanceName).Where(c => c.DatabaseName.DatabaseId.StartsWith("my-db-")).ToList();
+            var databasesToDelete = new List<string>();
+            // delete all the databases created before 5 hrs.
+            var timestamp = DateTime.UtcNow.AddHours(-5).ToTimestamp().Seconds;

```

Comment:
In Node.js we made this 4 hours.

```suggestion
            var timestamp = DateTime.UtcNow.AddHours(-4).ToTimestamp().Seconds;

```